src/contrast/pages/UploadNewPaper/ExperimentsSection/Findings.jsx

```diff
@@ -97,9 +97,7 @@ export default function Findings({
       return false;
     }
     if (field?.family == families["Temporal"]) {
-      return [field.onset, field?.offset].every(
-        (condition) => Boolean(condition) === true
-      );
+      return Boolean(field.onset) === Boolean(field.offset);
     }
 
     if (field?.family == families["Spatial Areas"] && field?.technique == 4) {
```

Comment:
I don't like your hardcoding a technique number, resolve it from the list of techniques you get from the server by name I think

Reply:
I get it as int...
![image](https://github.com/user-attachments/assets/af4f79cf-5851-480f-b32a-a83b42e40eb8)